Index: tests/keycloaktests/keycloack_auth_test.py
===================================================================
diff --git a/tests/keycloaktests/keycloack_auth_test.py b/tests/keycloaktests/keycloack_auth_test.py
--- a/tests/keycloaktests/keycloack_auth_test.py
+++ b/tests/keycloaktests/keycloack_auth_test.py
@@ -5,6 +5,7 @@
 from pages.keycloack_auth_page import KeycloackAuthForm
 from pages.controlpanelpages.topbar_cp_page import TopBarCpPage

+
 config = configparser.ConfigParser()
 config.read('ini_config/config.ini')

